cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/type/DatasetTypeManager.java

```diff
@@ -96,6 +102,26 @@ protected void shutDown() throws Exception {
     // do nothing
   }
 
+  private Map<String, DatasetModule> getExtensionModules(CConfiguration cConf) {
+    Map<String, DatasetModule> modules = Maps.newLinkedHashMap();
```

Comment:
also will fix other places in this module where map/set creation uses incorrect syntax.